core/src/parser/lexer.rs

```diff
@@ -426,7 +426,7 @@ pub enum MultiStringToken<'input> {
     /// variable number of `%` character, so the lexer matches candidate end delimiter, compare the
     /// number of characters, and either emit the `End` token above, or turn the `CandidateEnd` to a
     /// `FalseEnd` otherwise
-    #[regex("\"%+")]
+    #[regex("\"%+")] // m%%""%%{x}"%%m
```

Comment:
Is this comment left-over from some experiments? It seems to use the old, symmetric delimiters.

Reply:
Ah, probably.